<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmOption_4</class>
 <widget class="QWidget" name="frmOption_4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>539</width>
    <height>308</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>frmOption_4</string>
  </property>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>40</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>弹出提示框</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下单成功:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>声音提示:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>40</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>80</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>声音提示:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>80</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>弹出提示框</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>下单失败:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>挂单成交:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>120</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>弹出提示框</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_3">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>120</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>110</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>声音提示:</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>481</width>
     <height>141</height>
    </rect>
   </property>
   <property name="title">
    <string>下单反馈</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>声音提示:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBox_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>240</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>弹出提示框</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>481</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>撤单反馈</string>
   </property>
   <widget class="QComboBox" name="comboBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>30</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>200</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>弹出提示框</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>撤单成功:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>声音提示:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>撤单失败:</string>
   </property>
  </widget>
  <zorder>groupBox</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>checkBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>comboBox</zorder>
  <zorder>comboBox_2</zorder>
  <zorder>label_3</zorder>
  <zorder>checkBox_2</zorder>
  <zorder>label_4</zorder>
  <zorder>label_5</zorder>
  <zorder>checkBox_3</zorder>
  <zorder>comboBox_3</zorder>
  <zorder>label_6</zorder>
  <zorder>label_7</zorder>
  <zorder>checkBox_5</zorder>
  <zorder>checkBox_4</zorder>
  <zorder>label_10</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>frmOption_4</receiver>
   <slot>slotChange1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>372</x>
     <y>52</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>276</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_2</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>frmOption_4</receiver>
   <slot>slotChange2(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>399</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>52</x>
     <y>282</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_3</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>frmOption_4</receiver>
   <slot>slotChange3(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>433</x>
     <y>135</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>286</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_4</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>frmOption_4</receiver>
   <slot>slotChange4(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>452</x>
     <y>211</y>
    </hint>
    <hint type="destinationlabel">
     <x>303</x>
     <y>291</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBox_5</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>frmOption_4</receiver>
   <slot>slotChange5(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>446</x>
     <y>252</y>
    </hint>
    <hint type="destinationlabel">
     <x>482</x>
     <y>297</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotChange1(QString)</slot>
  <slot>slotChange2(QString)</slot>
  <slot>slotChange3(QString)</slot>
  <slot>slotChange4(QString)</slot>
  <slot>slotChange5(QString)</slot>
 </slots>
</ui>
